Week8AssignmentNew/ConsoleApp5/HomePage1.cs

```diff
@@ -0,0 +1,260 @@
+
+        //Display the product cost
+        string addedProductsCost = "((//div[contains(@class,'PaJLWc')])[{0}]/div[1]/span[contains(text(),'₹')])";
+
+        String nameOfProduct;
+        String priceOfProduct;
+        int itemPrice;
+        string NumberOfItems;
+        Boolean boolResult;
+        public int itemsCount;
+
+        public void EnterSearchItem()
+        {
+            Thread.Sleep(2000);
+            driver.FindElement(textBox).SendKeys("iphone 6");
+            driver.FindElement(textBox).SendKeys(Keys.Enter);
+        }
+
+        public void SelectMinDropdownElement()
+        {
+            java.lang.Thread.sleep(1000);
+            SelectElement MinimumValue = new SelectElement(driver.FindElement(minDropdownElement));
+            MinimumValue.SelectByValue("Min");
+        }
+        public void SelectMaxDropdownElement()
+        {
+            SelectElement MaximunValue = new SelectElement(driver.FindElement(maxDropdownElement));
+            MaximunValue.SelectByValue("50000+");
+        }
+
+        public void SettingAvailibilityFilter()
+        {
+            java.lang.Thread.sleep(1000);
+            driver.FindElement(availability).Click();
+        }
+
+        public void SettingExcludeOutOfStockFilter()
+        {
+            java.lang.Thread.sleep(1000);
+            driver.FindElement(excludeOutOfStockFilter).Click();
+        }
+
+        public int GetNumberOfPages()
+        {
+            try
+            {
+                Boolean disp = !(driver.FindElement(pageNumber).Displayed);
+            }
+            catch (Exception e)
+            {
+                return 1;
+            }
+
+            if (driver.FindElement(pageNumber).Displayed)
+            {
+                String textPages = driver.FindElement(pageNumber).Text;
+                string[] textPagesFooter = textPages.Split(' ');
+                int Pages = short.Parse(textPagesFooter[3]);
+                return (Pages);
+            }
+            else
+            {
+                return 1;
+            }
+        }
+
+        public int DiplayTotalItemsPresent()
+        {
+            String TotalItems = driver.FindElement(this.totalItems).Text;
+            String[] Totalitems = TotalItems.Split(' ');
+            int NumberOfItemsOnPage = Int16.Parse(Totalitems[3]);
+            return (NumberOfItemsOnPage);
+        }
+        public bool ItemUnavailable(int index)
+        {
+            try
+            {
+                return driver.FindElement(By.XPath(string.Format(unavailableItem, index))).Displayed;
+            }
+            catch (Exception e)
+            {
+                return false;
+            }
+        }
+        internal void AddToCart()
+        {
+            int pageCounter = 1;
+            int ppageCount = GetNumberOfPages();            
+            WebDriverWait wait = new WebDriverWait(driver, TimeSpan.FromSeconds(30));
```

Comment:
didn't see this wait utilised anywhere in this method

Reply:
I had made changes already.But the older file was read.I have deleted the older file.